core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/buffer_v1.go

```diff
@@ -108,6 +168,16 @@ func (b *logBuffer) Enqueue(uid *big.Int, logs ...logpoller.Log) (int, int) {
 	return buf.enqueue(blockThreshold, logs...)
 }
 
+func (b *logBuffer) evictReorgdLogs(reorgBlocks map[int64]bool) {
```

Comment:
q. do we know if we'll get the logs for the new reorged blocks enqueued again? If we are depending on Enqueue() to do that, then potentially we can just evict reorged logs at time of enqueue